readme.md

```diff
@@ -42,6 +42,8 @@ $ npm install --global awesome-lint
 
 Add it as a `test` script in package.json and activate Travis CI to lint on new commits and pull requests.
 
+Because Travis CI only clones with `depth=50` by default, which may result in wrong `git-repo-age` error, please set depth to `false` in `.travis.yml` if it needs.
```

Comment:
```suggestion
**Note:** [Travis CI only clones repositories to a maximum of 50 commits by default](https://docs.travis-ci.com/user/customizing-the-build/#git-clone-depth), which may result in a false positive of `awesome/git-repo-age`, and so you should set `depth` to `false` in `.travis.yml` if needed.
```

Reply:
@itaisteinherz Thanks for your professional English revise. And I have a little question about  `and so`. Shall we only use one of them?

- Travis CI is good, **and** we like it.
- Travis CI is good, **so** we like it.
- Travis CI is good, **and so** we like it.

Reply:
I prefer the third option because it sounds the most natural and logical to me.
Looking at [the definition of `and so`](https://www.ldoceonline.com/dictionary/and-so), it seems like it can be used similarly to `so`, but I feel like using it makes the logical transition clearer. `and` shouldn't be used here as it only points out the fact that the two parts of the sentence are both true, but not that a logical transition leads from the first part to the second.